tools/accuracy_checker/accuracy_checker/postprocessor/shift.py

```diff
@@ -0,0 +1,58 @@
+"""
+Copyright (c) 2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from ..config import NumberField
+from .postprocessor import PostprocessorWithSpecificTargets
+from ..representation import SegmentationAnnotation, SegmentationPrediction
+
+
+class Shift(PostprocessorWithSpecificTargets):
+    """
+    Shift prediction segmentation mask or annotation segmentation mask.
+    """
+
+    __provider__ = 'shift'
+
+    annotation_types = (SegmentationAnnotation, )
+    prediction_types = (SegmentationPrediction, )
+    
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'shift_value': NumberField(
+                optional=True, value_type=int, default=0, description="Value for shift."
+            )
+        })
+        return parameters
+    
+    def configure(self):
+        self.shift_value = self.config.get('shift_value')
+    
+    def process_image(self, annotation, prediction):
+
+        for annotation_ in annotation:
+            mask = annotation_.mask.astype(int)
+            update_mask = mask + self.shift_value
+            annotation_.mask = update_mask.astype(np.int16)
```

Comment:
why int16?

Reply:
Range int8 not big enough to cover all classes. Example: ADE20k with 150(151) classes.

Reply:
int8, yes, usually in segmentation used uint8 (0, 255), which enough should be enough (if of course you do not use negative values). int16, can be a problem for some specific postprocessing like resizing. but my question was more about astype uasge 2 line above with int and this, why they are different?

Reply:
Created this operation by analogy with encode_segmentation_mask, where used int for reading mask and int8 for result mask. But in this case we can't use int8 or uint8, because max in int8 < 151 and if use shift with negative value (example: 0 - 'null' label in ADE20k and shift -1 to annotation) and use uint8, we get error. So I am liable to use int16 as minimal  suitable type. But if this is not so necessary, i can delete astype and use usually int.

Reply:
then ok